draft-ietf-webpush-protocol.xml

```diff
@@ -938,6 +938,11 @@ Host: push.example.net
           then the push service MUST return a 410 (Gone) response to the application server
           monitoring the receipt subscription.
         </t>
+        <t><list style="hanging"><t hangText="Note:">
+          An application server also receives the push message resource URI when it sends
+          a push message.  This allows an application server to acknowledge messages it

```

Comment:
By acknowledge, you mean DELETE - so this is really a feature to delete in-flight messages? I think that calling it an acknowledgement is confusing since it could suggest that then the PS is supposed to send an ACK back to the AS as if the AS were the UA.